### DISCUSSION TITLE
Welcome, Mayank Jaiswal!

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#408] [Subscribe]
Ahoy @mayankjaiswal1709, welcome aboard!

Thank you for joining us! Every day there are new quiz questions posted in [Discussions](https://github.com/ParanoidUser/yolo/discussions/categories/galaxy-trivia), you can try to answer them to earn the Galaxy Brain badge. Spread the word, collaborate and have fun, have fun, have fun! 😊

Let me know if you have any questions.

**Comment by mayankjaiswal1709** [comment]
frist of all thankyou for joining me , yaa sure I will defiantly give the answers of quiz questions

**Comment by mayankjaiswal1709** [comment]
no questions !!

**Reply by ParanoidUser** [reply]
They are coming in 6 minutes! ;)

**Reply by mayankjaiswal1709** [reply]
ohky
